packages/react/src/components/media/players/VideoPlayer.tsx

```diff
@@ -118,6 +120,9 @@ export const VideoPlayer = React.forwardRef<HTMLVideoElement, VideoPlayerProps>(
           },
           {
             autoplay: autoPlay,
+            xhrSetup(xhr, _url) {
+              xhr.withCredentials = Boolean(allowCrossOriginCredentials);
+            },
```

Comment:
WDYT of only setting this if the prop is set?
```suggestion
            xhrSetup: !allowCrossOriginCredentials : undefined : (xhr, _url) => {
              xhr.withCredentials = true
            },
```